### PR TITLE
Change cli arguments parser to kingpin, add short commands

### LINKED ISSUE #49: Support for short flags
Right now we just support long flag format (ie `-port`, `-config`, etc). It would be nice to provide the short version for: `port`, `config` and `version`.

https://github.com/alecthomas/kingpin looks like a great candidate for this, but any other should be fine.

### PR DESCRIPTION
**Pull request by sponomarev**
Closes #49

### CODE REVIEW

File: cmd/httplab/main.go [hunk 1]
```diff
@@ -45,38 +45,23 @@ func defaultConfigPath() string {
...
-
 func main() {
 	var (
-		port    int
-		config  string
-		version bool
-		cors    bool
+		config   = kingpin.Flag("config", "Specifies custom config path.").Short('c').String()
+		cors     = kingpin.Flag("cors", "Enable CORS").Default("false").Bool()
+		port     = kingpin.Flag("port", "Specifies the port where HTTPLab will bind to.").Default("false").Default("10080").Int()
```
Comment (gchaincl) [inline cmd/httplab/main.go]: can u add .Short('p') here?

File: cmd/httplab/main.go [hunk 2]
```diff
@@ -45,38 +45,23 @@ func defaultConfigPath() string {
...
+		bindings = kingpin.Flag("bindings", "Show keyboard bindings").Bool()
```
Comment (gchaincl) [inline cmd/httplab/main.go]: shouldn't we keep `-h` for this one?

**Comment by sponomarev** [comment]
@gchaincl Done! Please, take a look!

**Comment by gchaincl** [comment]
@sponomarev thanks for this, but I'm closing this in favour of #50, it kept the original behavior which was to display the bindings information along the help.

**Pull request closed without merging**